aspnet/dnx/overview.rst

```diff
@@ -81,16 +81,16 @@ Dependencies in DNX consist of a name and a version number. Version numbers shou
     }
 
 
-The global.json file also defines the minimum DNX version ("sdk" version) needed to build the project.
+The *global.json* file also defines the minimum DNX version ("sdk" version) needed to build the project.
 
 Dependencies are transitive in that you only need to specify your top level dependencies. DNX will handle resolving the entire dependency graph for you using the installed NuGet packages. Project references are resolved at runtime by building the referenced project in memory. This means you have the full flexibility to deploy your DNX application as package binaries or as source code.
 
 .. _dnx-concept-packages-and-feeds:
 
 Packages and feeds
-------------------
+---------------------------
 
-For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project, like this::
+For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project. The following command downloads and installs all packages that are listed in the *project.json* file:
```

Comment:
That would be better because when I try to sync the update I get the following error
[cid:image001.png@01D0A445.7C5EF140]


From: Daniel Roth [mailto:notifications@github.com]
Sent: Thursday, June 11, 2015 12:51 PM
To: aspnet/Docs
Cc: Rick Anderson
Subject: Re: [Docs] Minor typos (#301)


In aspnet/dnx/overview.rst<https://github.com/aspnet/Docs/pull/301#discussion_r32260414>:

>

> -For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project, like this::

> +For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project. The following command downloads and installs all packages that are listed in the *project.json* file:

Actually, looks like you might be able to just update the file on the branch using the online editor.

—
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Docs/pull/301/files#r32260414>.